jupyter_book_to_htmlbook/xref_processing.py

```diff
@@ -36,3 +37,29 @@ def process_interal_refs(chapter):
             ref.string = f'#{uri}'
             ref['href'] = f'#{uri}'
     return chapter
+
+
+def process_ids(chapter, existing_ids=[]):
```

Comment:
Praise: This is robust and simple solution to a problem that plagues all book conversion tools!